src/StripeTests/Services/Accounts/AccountServiceTest.cs

```diff
@@ -108,20 +111,23 @@ public async Task CreateAsync()
         public void Delete()
         {
             var deleted = this.service.Delete(AccountId);
+            this.AssertRequest(HttpMethod.Delete, "/v1/accounts/acct_123");
```

Comment:
minor but using `AccountId` might be cleaner (not convinced it would)

Reply:
I like the explicitness of using the expected string directly, without interpolation. It's also consistent with what we do in other libraries (previously, in some libraries, we _had_ to use interpolated strings because stripe-mock would return objects with a different ID than the requested one, but 1/ the stripe-mock issue is fixed now and 2/ this doesn't apply to stripe-dotnet in any case as it uses static methods on service objects, not instance methods on resource objects).